husarion_ugv_lights/test/unit/led_components/test_led_segment.cpp

```diff
@@ -150,34 +146,33 @@ TEST(TestLEDSegmentInitialization, FirstLedPosition)
 
 TEST_F(TestLEDSegment, GetAnimationFrameNoAnimation)
 {
-  EXPECT_TRUE(husarion_ugv_utils::test_utils::IsMessageThrown<std::runtime_error>(
-    [&]() { led_segment_->GetAnimationFrame(); }, "Segment animation not defined"));
+  EXPECT_FALSE(husarion_ugv_utils::test_utils::IsMessageThrown<std::runtime_error>(
+    [&]() { led_segment_->GetAnimationFrame(); },
+    "Segment animation not defined"));  // with the current implementation, this should not throw an
+                                        // error
 }
```

Comment:
Why we are having this test if it doesn't test anything. What frame will be returned if no animation is defined for led segment?